src/components/add-button/index.tsx

```diff
@@ -6,41 +6,47 @@ import { Plus } from 'react-feather';
 import { getSpace, SpaceSize } from '../space';
 import styled from 'styled-components';
 
-export interface AddPageButtonProps {
+export interface AddButtonProps {
 	onClick?: React.MouseEventHandler<HTMLElement>;
+	text: string;
+	margin?: boolean;
 }
 
-const StyledAddPageButton = styled.button`
+const StyledAddButton = styled.button`
 	position: relative;
 	box-sizing: border-box;
 	height: 60px;
 	width: 100%;
-	border: 1px solid ${Color.Grey80};
+	border: 1px solid ${Color.Grey90};
 	border-radius: 6px;
 	background-color: transparent;
-	margin: ${getSpace(SpaceSize.XS)}px;
+	margin: ${(props: AddButtonProps) => (props.margin ? `${getSpace(SpaceSize.XS)}px` : '0')};
```

Comment:
Could the decision wether `margin` should be used placed outside the component via `<Space/>`?